contracts/PurchasabilityValidator.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity 0.5.7;
+
+import "./DrugItem.sol";
+
+contract PurchasabilityValidator {
+
+    uint8 public VALID_FOR_PURCHASE = 100;
+
+    function isPurchasable(DrugItem _drugItem) public view returns (uint8[] memory) {
+        uint8[] memory result = new uint8[](10);
+        result[0] = VALID_FOR_PURCHASE;
+        return result;
```

Comment:
you can use `public view return (uint8[] memory result)` and then skip the last line in the method ;)